examples/allennlp_simple.py

```diff
@@ -0,0 +1,127 @@
+import os
+import shutil
+
+import allennlp
+import allennlp.data
+import allennlp.models
+import allennlp.modules
+import torch
+
+import optuna
+
+
+DEVICE = -1  # If you want to use GPU, use DEVICE = 0.
+MAX_DATA_SIZE = 3000
+
+DIR = os.getcwd()
+MODEL_DIR = os.path.join(DIR, 'result')
+
+GLOVE_FILE_PATH = (
+    'https://s3-us-west-2.amazonaws.com/allennlp/datasets/glove/glove.6B.50d.txt.gz'
```

Comment:
Sorry I'm not familiar with AllenNLP but is it the case that this AWS URL (and the ones below) is officially documented somewhere?

Reply:
This example is based on [the allentune's official example](https://github.com/allenai/allentune/tree/691212f7ede331ba314f5f2f1b0c2fb9c7e71b8e/examples) and this AWS URL is used [here](https://github.com/allenai/allentune/blob/691212f7ede331ba314f5f2f1b0c2fb9c7e71b8e/examples/classifier.jsonnet#L45).